docker-compose.dev.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+volumes:
+  db_dev:
+  mosquitto-conf_dev:
+  mosquitto-data_dev:
+services:
+  db:
+    image: postgres:alpine
+    environment: 
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: teslamate_dev
+      POSTGRES_HOST_AUTH_METHOD: trust
+    volumes:
+      - db_dev:/var/lib/postgresql/data 
+    
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    depends_on: 
+      - db
+      - mqtt
+    environment: 
+      MIX_ENV: dev
+      DATABASE_HOST: db
+      MQTT_HOST: mqtt
+    ports:
+      - 4000:4000
+    volumes:
+      - .:/app
```

Comment:
If the local directory is used as volume for the "/app" directory, then the installed Elixir/npm dependencies will be lost, won't they? With a fresh repo it fails for this reason.

Reply:
Yea, you are right! I changed it, but you now have to set the correct permission by supplying your UID and GID.